hack/lib/test.sh

```diff
@@ -223,3 +228,20 @@ kube::test::if_has_string() {
     return 1
   fi
 }
+
+# Returns true if the required resource is part of supported resources.
+# Expects env vars:
+#   SUPPORTED_RESOURCES: Array of all resources supported by the apiserver. (*)
+#   means it supports all resources.
+#   $1: Name of the resource to be tested.
+kube::test::if_supports_resource() {
```

Comment:
Good catch. Fixed the issue and also added a check to ensure that runTests does not complete without running any tests. It does not check that all tests were run but verifies that atleast a few were run.